### PR TITLE
Feature/liq03 api spec

### PR DESCRIPTION
**Pull request by gshivach**
added spec for 3 endpoints for LIQ03 progress-report
Post
Get
Delete

Also amended spec for Resolution and SOA

with success and error responses.

### CODE REVIEW

**Review by randerson123 — APPROVED** [review]
Approved

**Review by randerson123 — APPROVED** [review]
Approved

File: apispec/schema.yml [hunk 1]
```diff
@@ -423,6 +423,93 @@ paths:
                     type: string
                     format: date
 
+  /transactions/{transaction_id}/insolvency/progress-report:
+    post:
+      tags:
+        - "progress report"
```
Comment (macreman) [inline apispec/schema.yml:429]: Tags should be capitalised for consistency i.e. "Progress Report". Same for both other instances below

File: apispec/schema.yml [hunk 2]
```diff
@@ -680,6 +768,45 @@ components:
               type: string
               format: uri
               example: /transactions/{transaction_id}/insolvency/statement-of-affairs
+
+    ProgressReportWritable:
+      type: object
+      properties:
+        progress_report_from_date:
```
Comment (macreman) [inline apispec/schema.yml:775]: Given the context of the fields being in a progress report resource I think these date fields can be shortened to just "from_date" and "to_date". Same for the non-writable resource below too

File: apispec/schema.yml (hunk 1 above)
Comment (gshivach) [reply apispec/schema.yml]: fixed

File: apispec/schema.yml (hunk 2 above)
Comment (gshivach) [reply apispec/schema.yml]: fixed

**Pull request merged**